### DISCUSSION TITLE
Can't figure out Record Selector and moving data from json response into records

### DISCUSSION THREAD

**Discussion by WelcomeCIT** [airbytehq/airbyte#40728] [Connector Builder]
I have an awkward API that returns data in a JSON style but all the data is contained in the 'body' in a csv style. My 'Records' section on the test Export shows 0 records and i can't figure out how to 1: extract the data from the body and put it into the records and 2: if its even possible to parse the csv style data because it can be quite large. Im assuming its something to do with Record Selector but could someone point me in the right path? I've tried entering **body** in the field path but i get an error **No records could be extracted from responses, make sure the record selector is set correctly.**

[
![Screenshot 2024-07-04 071647](https://github.com/airbytehq/airbyte/assets/174683883/ae57843e-bc9d-4b92-bde1-5d21156ea0ad)
](url)

[
![Screenshot 2024-07-04 071713](https://github.com/airbytehq/airbyte/assets/174683883/cef0fb3e-ed92-4275-bbf5-4eeb9abf2f65)
](url)

**Comment by Pedrxx** [comment]
Hello. Did you solve it ??
